src/sources/journald.rs

```diff
@@ -58,10 +58,11 @@ enum BuildError {
     #[snafu(display("Cannot use both `units` and `include_units`"))]
     BothUnitsAndIncludeUnits,
     #[snafu(display(
-        "The unit {:?} is duplicated in both include_units and exclude_units",
-        unit
+        "The Journal field/value pair {:?}:{:?} is duplicated in both include_matches and exclude_matches (or include_units and exclude_units).",
+        field,
+        value,
```

Comment:
Since there are separate units and matches options, I think this and the one below would be better split into a separate error for each.

Reply:
Hmm, I was thinking those errors would be caught separately, and so distinct messages would help the user. Oh well.

Reply:
Thank you for pointing out the issue!

It's certainly more friendly to separate them.
The above case is fine if expressed with DuplicatedMatches Error, so I fixed it. f61dad3